contrib/aws/vpc/variables.tf

```diff
@@ -0,0 +1,54 @@
+variable vpc_cidr{
+    type=string
+    default = "10.0.0.0/16"
+    validation {
+      condition     = contains(["10.0.0.0/16", "11.0.0.0/16", "12.0.0.0/16"], var.vpc_cidr)
+      error_message = "VPC CIDR limited to: 10.0.0.0/16 11.0.0.0/16 12.0.0.0/16"
+    }
+    
+}
+variable multi_az{
+    type = bool
+    default = false
+}
+variable "aws_region" {
+  type        = string
+  description = "The region to create the ROSA cluster in"
+  
+  validation {
+    condition     = contains(["eu-central-1", "eu-west-1", "us-east-1", "us-east-2", "us-west-2"], var.aws_region)
+    error_message = "HyperShift is currently only availble in these regions: eu-central-1, eu-west-1, us-east-1, us-east-2, us-west-2."
```

Comment:
tf provider doesn't currently support Hypershift clusters.
Should we use only regions where Hypershift is supported?

Reply:
Removed

Reply:
still can see the hypershift validation (limitation)

Reply:
Fixed